Assets/popUpObject copy

```diff
@@ -0,0 +1,131 @@
+const width = window.innerWidth;
+/* eslint-disable no-unused-vars */
+const projectsInfo = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
```

Comment:
- [ ] Kindly increase the size of the card as well as the text description.
<img width="1222" alt="Screenshot 2022-12-23 at 08 19 32" src="https://user-images.githubusercontent.com/81444338/209368201-c2f232d9-0f56-4010-bf4d-2c4efac666e3.png">